### DISCUSSION TITLE
Welcome to Ladder99 Discussions!

### DISCUSSION THREAD

**Discussion by bburns** [Ladder99/ladder99-ce#1] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by fastxl** [comment]
I am a bit of a novice and just poking around to see how things work. How can I get the postgres password so I can connect with an SQL browser?
